emu_mps/mps_backend_impl.py

```diff
@@ -118,8 +120,17 @@ def __init__(self, mps_config: MPSConfig, pulser_data: PulserData):
         self.timestep_count: int = self.omega.shape[0]
         self.has_lindblad_noise = pulser_data.has_lindblad_noise
         self.lindblad_noise = torch.zeros(2, 2, dtype=torch.complex128)
-        self.full_interaction_matrix = pulser_data.full_interaction_matrix
-        self.masked_interaction_matrix = pulser_data.masked_interaction_matrix
+        self.qubit_permutation = (
+            optimat.minimize_bandwidth(pulser_data.full_interaction_matrix)
```

Comment:
please also add this to the qubit_order field in the `Results`. Currently, it's left empty, but this is really an error on our side.

Reply:
shame on me, didn't notice

Reply:
done